cms/templates/js/unit-access-editor.underscore

```diff
@@ -0,0 +1,110 @@
+<%
+var groupAccess = xblockInfo.get('group_access');
```

Comment:
Please try moving (renaming) the file instead of deleting the old file and creating a new one. The move operation will retain git history, and provide a diff of what has actually changed.